src/index.js

```diff
@@ -1,7 +1,4 @@
-import 'babel-polyfill';
-import React from 'react';
-import { render } from 'react-dom';
+import { hot } from 'react-hot-loader';
+import AppContainer from './AppContainer';
 
-import StripesCore from './App';
-
-render(<StripesCore />, document.getElementById('root'));
+export default hot(module)(AppContainer);
```

Comment:
I'm surprised that this works. `AppContainer` doesn't export anything for this to hook into.

I think it would be a little more explicit if we had a `HotApp` module we import into the index and render instead of the `AppContainer` file. Especially because the naming `AppContainer` suggests it is a component, but that file actually renders the app.

Reply:
Yeah, that change makes sense. Commit added to move the render back to `index.js`, create a `HotApp`, and remove `AppContainer`.